test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -122,61 +115,37 @@ public Task Localization_CustomCulture()
                 "Hello Hello Hello Hello");
         }
 
-        [ConditionalFact]
```

Comment:
Appears a few tests went away completely and some test pairs switched `RuntimeFlavor` order. But it's not clear.